tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/op/core/ShapeOps.java

```diff
@@ -49,28 +42,30 @@
     /**
      * Flatten the operand to 1 dimension
      *
-     * @param <T> the type of operand
-     * @param scope current scope
+     * @param <T>     the type of operand
+     * @param scope   current scope
      * @param operand the operand to flatten
      * @return the reshaped operand
      */
     @Endpoint(name = "flatten")
-    public static <T extends TType> Operand<T> flatten(Scope scope, Operand<T> operand) {
+    public static <T extends TType> Operand<T> flatten(
+            Scope scope, Operand<T> operand) {
```

Comment:
Just double-checking with you @JimClarke5 that you are using Google Java Formatter in your IDE, as there are a lot of reformatting changes in the PR (quickly like that, it looks like the previous version of the code was closer to what we find in the other source files).

Reply:
I am using IntelliJ which seemed to have its own mind about formatting. I will see if I can add the Google Java Formatter to it.

Reply:
Ok, if found the google-java-format plugin for intellij, so I can push up the reformatted code, pending  resolution to the first 2 questions.